ccvm_simulators/tests/unit/problem_classes/test_problem_instance.py

```diff
@@ -9,8 +9,8 @@ class TestProblemInstance(TestCase):
     def setUp(self):
         self.device = "cpu"
         self.instance_type = "tuning"
-        self.basepath = os.path.dirname(__file__)
-        self.file_path = self.basepath + "/test_instances/test020-100-10.in"
+        self.test_folder_path = "ccvm_simulators/tests/data/test_instances"
```

Comment:
Does this assume that you are running pytest from a specific directory?

Reply:
Got call. I've fixed it, so it doesn't depend on the specific directory from which the tests are executed.